discord_slash/client.py

```diff
@@ -600,7 +606,8 @@ async def _group_kick_user(ctx, user):
         subcommand_group_description = subcommand_group_description or sub_group_desc
 
         def wrapper(cmd):
-            obj = self.add_subcommand(cmd, base, subcommand_group, name, description, base_description, subcommand_group_description, guild_ids, options, connector)
+            obj = self.add_subcommand(cmd, base, subcommand_group, name, description, base_description,
+                                      subcommand_group_description, guild_ids, options, connector)
```

Comment:
Why has this changed other than formatting (which probably should happen, but be in a different pr), there are no other changes here :upside_down_face:

Reply:
`Ctrl Alt + L` in pycharm reformats your code, I use it a lot when coding, I can revert it, but technically it abides by PEP8 now

Reply:
PEP8 isn't really a concern to me IMO, code readability > formatting.